benchmarks/dbscan/dbscan.cpp

```diff
@@ -50,6 +50,41 @@ int getDataDimension(std::string const &filename, bool binary)
   return dim;
 }
 
+std::string backendsList(std::string const &delim = "|")
```

Comment:
Why do we need a defaulted argument here?
```suggestion
std::string backendsList()
```

Reply:
Don't really need it. Just thought that if we decided it a general function to be used in other place, may as well allow some customization.